tests/neo-vm.Tests/UtReferenceCounter.cs

```diff
@@ -106,13 +109,14 @@ public void TestCircularReferences()
         public void TestRemoveReferrer()
         {
             using ScriptBuilder sb = new ScriptBuilder();
-            sb.EmitPush(0); //{0}{}:1
-            sb.Emit(OpCode.NEWARRAY); //{A[]}|{}:1
-            sb.Emit(OpCode.DUP); //{A[],A[]}|{}:2
-            sb.EmitPush(0); //{A[],A[],0}|{}:3
-            sb.Emit(OpCode.NEWARRAY); //{A[],A[],B[]}|{}:3
-            sb.Emit(OpCode.TOALTSTACK); //{A[],A[]}|{B[]}:3
-            sb.Emit(OpCode.DUPFROMALTSTACK); //{A[],A[],B[]}|{B[]}:4
+            sb.Emit(OpCode.INITSSLOT, new byte[] { 1 }); //{}|{null}:1
+            sb.EmitPush(0); //{0}|{null}:2
+            sb.Emit(OpCode.NEWARRAY); //{A[]}|{null}:2
+            sb.Emit(OpCode.DUP); //{A[],A[]}|{null}:3
+            sb.EmitPush(0); //{A[],A[],0}|{null}:4
+            sb.Emit(OpCode.NEWARRAY); //{A[],A[],B[]}|{null}:4
+            sb.Emit(OpCode.STSFLD0); //{A[],A[]}|{B[]}:3
+            sb.Emit(OpCode.LDSFLD0); //{A[],A[],B[]}|{B[]}:4
```

Comment:
Could be Argument array as well I guess, as Static is more for "bottom" behavior... but that's fine, it's the same now.